src/main/kotlin/no/nav/familie/tilbake/oppgave/OppgaveService.kt

```diff
@@ -0,0 +1,120 @@
+package no.nav.familie.tilbake.oppgave
+
+import io.micrometer.core.instrument.Counter
+import io.micrometer.core.instrument.Metrics
+import no.nav.familie.kontrakter.felles.oppgave.Behandlingstema
+import no.nav.familie.kontrakter.felles.oppgave.FinnOppgaveRequest
+import no.nav.familie.kontrakter.felles.oppgave.IdentGruppe
+import no.nav.familie.kontrakter.felles.oppgave.Oppgave
+import no.nav.familie.kontrakter.felles.oppgave.OppgaveIdentV2
+import no.nav.familie.kontrakter.felles.oppgave.OppgaveResponse
+import no.nav.familie.kontrakter.felles.oppgave.Oppgavetype
+import no.nav.familie.kontrakter.felles.oppgave.OpprettOppgaveRequest
+import no.nav.familie.kontrakter.felles.oppgave.Tema
+import no.nav.familie.kontrakter.felles.tilbakekreving.Ytelsestype
+import no.nav.familie.tilbake.behandling.BehandlingRepository
+import no.nav.familie.tilbake.behandling.FagsakRepository
+import no.nav.familie.tilbake.common.repository.findByIdOrThrow
+import no.nav.familie.tilbake.integration.familie.IntegrasjonerClient
+import no.nav.familie.tilbake.person.PersonService
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+import java.util.UUID
+
+@Service
+class OppgaveService(private val behandlingRepository: BehandlingRepository,
+                     private val fagsakRepository: FagsakRepository,
+                     private val integrasjonerClient: IntegrasjonerClient,
+                     private val personService: PersonService) {
+
+
+    private val antallOppgaveTyper: MutableMap<Oppgavetype, Counter> = mutableMapOf()
```

Comment:
Jeg tror det er bedre hvis vi kan unngå å bruke mutableMap i klassenivå. Vi kan bruke vanlig map i stedet

Reply:
Kanskje 
private val antallOppgaveTyper = Oppgavetype.values().associateWith { 
        Metrics.counter("oppgave.opprettet", "type", it.name) 
    }

Reply:
Da kan du fjerne økTellerForAntallOppgaveTyper og helle bruke incremnet direkete fordi du slipper null-sjekken.

Reply:
den var snasen :)